src/main/resources/BridgeServer2.conf

```diff
@@ -149,9 +149,12 @@ prod.usersigned.consents.bucket = bridgepf-prod-awss3usersignedconsentsdownloadb
 # Bootstrap user for integration tests (will only be used when first initializing your database).
 # This should match the synapse.test.user.* credentials set for your integration test properties 
 # in your bridge-sdk-test.properties file.
-admin.email = dummy-value
+admin.email = bridgetest
```

Comment:
I think this is an email address, which could vary by developer.

Reply:
Yes, in my local file, local.admin.email is an email address. Are you saying change this to bridgetest@synapse.org (which I am guessing will also work)?

Reply:
I believe this is the email address of the Bridge account (eg dwayne.jeng@sagebase.org), so it would need to be overridden by each developer.